module/manifest.json

```diff
@@ -1,14 +1,15 @@
 {
-    "changes": "* Upgrade Trigger.io platform to `v2.8.6`\n* Update Parse iOS SDK to `1.17.2`\n* Update module to use Bolts v1.10.0 which replaces usage of UIWebView with WKWebView",
-    "dependencies": {
-        "bolts": {
-            "minimum_version": "1.10.0",
-            "maximum_version": "1.10.0"
-        }
-    },
-    "description": "Parse.com configuration to enable push notifications.\n\nThe parse module uses the Parse.com SDK to allow for native push notifications.",
-    "min_platform_version": "v2.8.6",
-    "namespace": "parse",
-    "platform_version": "v2.8.6",
-    "version": "3.4.0"
-}
+	"changes": "* Update Parse Android SDK to 1.20.0",
+	"dependencies": {
+		"bolts": {
+			"maximum_version": "1.10.0",
+			"minimum_version": "1.10.0",
+			"module": "bolts"
+		}
+	},
+	"description": "Parse.com configuration to enable push notifications.\n\nThe parse module uses the Parse.com SDK to allow for native push notifications.",
+	"min_platform_version": "v2.8.6",
+	"namespace": "parse",
+	"platform_version": "v2.8.6",
+	"version": "3.5.8"
```

Comment:
Why jumping from `3.4.0` to `3.5.8` and not to `3.5.0`?

Reply:
At least to my knowledge and from what I searched online, there is no way to try a module without pushing it. 
And everytime you change the source code, you need to push a new version. 

I experimented with setting Trigger platform to `2.9.0`, without realizing it wasn't part of the task's ACs. We'll have to do it sometime soon anyways. 

I saw it is a normal workflow, based on previous version uploads (latest minors were `3.3.14` or `3.1.9`)

Reply:
You can overwrite an existing version, but maybe only the latest one. This is at least how I did it in the past. But you are right it is then not possible anymore to go back.